resources/public/api/conf/1.0/application.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.3
 
 info:
   version: 1.0.0
-  title: Manage ISAs API
+  title: ISA Reporting API
```

Comment:
ISA Returns API 

That was the majority choice last week. Now recorded in Decision log.